includes/admin_import.php

```diff
@@ -183,14 +213,20 @@ function tsml_import_page()
 					'type' => 'JSON',
 				];
 
-				//import feed
-				tsml_import_buffer_set($body, $data_source_url, $data_source_parent_region_id);
+				if (count($feed_updates) === 0) {
+					//import feed
+					tsml_import_buffer_set($body, $data_source_url, $data_source_parent_region_id);
+				}
+				else {
```

Comment:
would be great to format this with intelephense

Reply:
as discussed, i'll handle formatting in release prep

Reply:
Check...